js/flightlog.js

```diff
@@ -450,8 +455,38 @@ function FlightLog(logData) {
                                 // TODO pending to do something with GPS frames
                                 // The frameValid can be false, when no GPS home (the G frames contains GPS position as diff of GPS Home position).
                                 // But other data from the G frame can be valid (time, num sats)
+
+                                //H Field G name:time,GPS_numSat,GPS_coord[0],GPS_coord[1],GPS_altitude,GPS_speed,GPS_ground_course
+                                for (let i = 0; i < frame.length; i++) {
+                                    lastGPS[i] = frame[i];
+                                }
```

Comment:
You are doing the same for H and G frames. If I'm not wrong, the H frame contains only two data (the coordinates), and G contains all the seven data (if GPS Home established). The loop works for both, but I suppose is better to store the fields into different variables because they are used in a different way.

Reply:
Thank you for the feedback and knowledgesharing!

Will look into these tonight!